sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/JdbcTableDataObject.scala

```diff
@@ -176,6 +202,10 @@ case class JdbcTableDataObject(override val id: DataObjectId,
   override def isDbExisting(implicit session: SparkSession): Boolean = connection.catalog.isDbExisting(table.db.get)
   override def isTableExisting(implicit session: SparkSession): Boolean = connection.catalog.isTableExisting(table.db.get, table.name)
 
+  def deleteTable(implicit session: SparkSession): Unit = {
```

Comment:
This name is a bit misleading: Sounds like deleting the table itself where in fact you only delete FROM table.

Reply:
Ok, fixed to deleteAllData